### ISSUE TITLE
caddy url / target url for app spawned by deployer

### ISSUE THREAD

**Issue by alexschroeter** [alexschroeter/apptainer-deployer#8]
Currently the URL of the server is hardcoded. This URL should be taken from the server deployer is connected to. I am not sure which service should provide this information (herre is my best guest, but @jhnnsrs should give his opinion).
This is related to https://github.com/jhnnsrs/deployer/issues/2 .

**Comment by jhnnsrs** [comment]
Agreed on it by principle. Even though it might not be always accurate (i.e when running in a separate docker network) maybe a ENV overwrite is a good idea.

```python
fakts = get_current_fakts()
x = await fakts.aget("lok.endpoint_url")
```
